docs/website/static/item-data.txt

```diff
@@ -0,0 +1,75 @@
+----------------------------------------------
+Generate html from this data via vim macros in item-template.txt
```

Comment:
Hah fun.

This is pretty close to how Hugo's [Data Templates](https://gohugo.io/templates/data-templates/) work. Would you think a little extra time here to replace the vim macros with a small code reorganization would be worth it?

I think it should be achievable to have _this_ data in a yaml file, that'll be sourced automatically by Hugo, and to have the template that generated the community page use that data: basically like the ecosystem page is built: https://github.com/open-policy-agent/opa/blob/main/docs/website/layouts/docs/ecosystem.html.html#L51-L52 this is the part that iterates over the data document, this yaml file: https://github.com/open-policy-agent/opa/blob/main/docs/website/data/integrations.yaml